src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/PlatformManager.java

```diff
@@ -156,12 +156,6 @@ public Payment getPlatformPayment(String name) throws PreActionException {
 
             //if not rethrow the exception and update the db
             if (throwable != null) {
-                try {
```

Comment:
as the description says:
** No more weak recovery (for more detail see commit)**